src/shared/telemetry/vscodeTelemetry.json

```diff
@@ -47,6 +58,14 @@
             "metadata": [
                 {
                     "type": "result"
+                },
+                {
+                    "type": "documentFormat",
+                    "required": false
```

Comment:
If this is not required why do we need the `"Unselected"` value ?

Reply:
leaving as undefined